package.json

```diff
@@ -45,6 +46,7 @@
     "ci-nightly-sampler": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --otel-url='' --iterations 1000 --min-wfs-per-sec 5 --max-cached-wfs 500 --workflow yummy-sampler-mode",
     "wait-namespace": "node ./scripts/wait-on-temporal.mjs",
     "lint": "eslint packages/*/src --ext .ts --no-error-on-unmatched-pattern && prettier --end-of-line auto --check .",
+    "lint.prune": "! ts-prune -p tsconfig.prune.json --ignore \"used in module\" | grep -v \".d.ts\"",
```

Comment:
Until https://github.com/nadeesha/ts-prune/issues/156 is resolved, `--skip` doesn't work. Hence the `| grep -v`. 

By default `grep` returns `0` if found and `1` or not. We actually want the opposite... so the `!` at the beginning inverts the exist code. 🧠

Reply:
Will this syntax work on windows?
Might be worth adding the prune CI step to the build-lint-test CI job to verify.

Reply:
Alas, it will not -- Windows also doesn't have `!` or `grep` 😞....

Reply:
Aha, the upstream issue's fixed was released!